src/Importer/ModuleImporter.ts

```diff
@@ -116,4 +108,78 @@ export class ModuleImporter implements ImporterContract {
       })
       .join('\n');
   }
+
+  protected transformScript(contents: string, resolveDir: string, sourcefile: string): string {
+    if (contents.includes('import ')) {
```

Comment:
Is it useful to use `transformSync` instead of `buildSync` when there is no `import` statement? If a user want to update a file to add an import line, `import ` will be detected and `buildSync` will be used, even if the actual preset doesn't import anything. Would that be an issue?

Reply:
Thank you for this review.

I added this test to reduce at its minimum the impact on existing presets.

IMHO, `buildSync` should be similar to `transformSync` when there is no `import` statement so it should work with existing presets but I wanted to be sure that I didn't introduce a breaking change.

If you want, I can try to remove this test so you can test if it works well in your existing presets.
Do you agree?

Reply:
I also think they would be similar. I don't think it's a breaking change and I'd rather have consistency over all types of preset. I'd be happy to test my presets after you make the change!

Reply:
I did the change.